tide/master/MasterApplication.cpp

```diff
@@ -229,6 +233,11 @@ void MasterApplication::_init()
         _planarController.reset(
             new PlanarController(_config->getPlanarSerialPort()));
     }
+    connect(_timer.get(), &PowerTimer::poweroff, [this]() {
```

Comment:
the connect needs to be in the:
if (!_config->getPlanarSerialPort().isEmpty()) {...}